pkg/server/status.go

```diff
@@ -1235,18 +1231,13 @@ func (s *statusServer) Stacks(
 
 	switch req.Type {
 	case serverpb.StacksType_GOROUTINE_STACKS:
-		bufSize := runtime.NumGoroutine() * stackTraceApproxSize
-		for {
-			buf := make([]byte, bufSize)
-			length := runtime.Stack(buf, true)
-			// If this wasn't large enough to accommodate the full set of
-			// stack traces, increase by 2 and try again.
-			if length == bufSize {
-				bufSize = bufSize * 2
-				continue
-			}
-			return &serverpb.JSONResponse{Data: buf[:length]}, nil
+		var buf bytes.Buffer
+
```

Comment:
I think we might want to default to `2` if req.Debug is 0, e.g. 
```
debug := 2
if req.Debug > 0 {
debug = req.Debug
}
```
So that if an older client, that doesn't set req.Debug at all, is asking for stacks from a new node, it gets what it expects (2)?